lib/lcov_ex/formatter.ex

```diff
@@ -9,38 +9,60 @@ defmodule LcovEx.Formatter do
   @type path :: binary()
   @type coverage_info :: {binary(), integer()}
 
+  @newline 10
```

Comment:
```suggestion
  @newline "\n"
```
I think this is more explicit.

Reply:
You're right; I was being too clever :)